src/main/java/http/routes/MethodOptions.java

```diff
@@ -15,7 +12,7 @@ public class MethodOptions implements Route {
 
     public MethodOptions() {
         CRLF = "\r\n";
-        headers = Arrays.asList("GET", "HEAD", "OPTIONS");
+        headers = Arrays.asList(Methods.GET.getMethod(), Methods.HEAD.getMethod(), Methods.OPTIONS.getMethod());
```

Comment:
refactor: There looks like a pattern is emerging here, where it will be a good idea to create a responseHeader class. But this refactor can wait till all the structured data work has been completed and before you move on to file server. Just have a think about how you could get rid of the duplicated code.

Reply:
Will look to refactor before adding the file server, thanks!